models/Element/ElementInterface.php

```diff
@@ -243,9 +243,13 @@ public function setVersionCount(?int $versionCount): self;
     public function getVersionCount(): int;
 
     /**
-     * @return $this
+     * @param array $parameters{
+     *      versionNote: string, # Used to set a note on the version
+     * } Associative array with optional parameters that are also available to Events
```

Comment:
```suggestion
     * @param array{
     *      versionNote: string, # Used to set a note on the version
     * } $parameters Associative array with optional parameters that are also available to Events
```

Reply:
Sure about this? PHP Storm complains with the error "Argument type does not match the declared. Doc tag without variable name doesn't provide type information for any expression" if I commit this change

Reply:
@NiklasBr Maybe the comment is not allowed in the type? Can we put it also to the description?

Reply:
That might be it.

Reply:
```suggestion
     * @param array{versionNote?: string} $parameters Associative array with optional parameters that are also available to Events
     *  - versionNote: Used to set a note on the version
```